src/etc/init.d/sdc-init

```diff
@@ -0,0 +1,29 @@
+#!/sbin/openrc-run
```

Comment:
Is there any reason we can't just continue to use rc.local like everything else?

Reply:
openrc runs the local scripts after initializing everything else. I wanted to have `/etc/resolve.conf` created and the zfs filesystems mounted before starting any services. I currently add the sdc-init script into the `boot` runlevel. I also though it would have a nicer UI if each step was printed:

```
# /etc/init.d/sdc-init start 2>/dev/null 
 * Loading SDC configuration ...
(info) - /lib/smartdc/mdata-fetch - Retrieving metadata user-data
(info) - /lib/smartdc/mdata-fetch - Metadata user-data not defined.
(info) - /lib/smartdc/mdata-fetch - Retrieving metadata user-script...
(info) - /lib/smartdc/mdata-fetch - Metadata user-script not defined.
(info) - /lib/smartdc/mdata-fetch - Retrieving metadata operator-script...
(info) - /lib/smartdc/mdata-fetch - Metadata operator-script not defined.                                                                                              [ ok ]
 * Setting provisioned state ...                                                                                                                                       [ ok ]
 * Mounting zfs datasets ...
(info) - /lib/smartdc/mount-zfs - Mounting all ZFS filesystems                                                                                                         [ ok ]
 * Generating resolv.conf ...                                                                                                                                          [ ok ]
 * Executing user scripts ...                            
 ```

I could just make sdc-init execute the rc.local to not duplicate the logic. Does that work?

Reply:
I'm now calling the rc script from sdc-init.